### PR TITLE
Add mxid and xid metrics

### PR DESCRIPTION
**Pull request by andreasscherman**

### CODE REVIEW

File: postgresql_metrics/metrics_gatherer.py [hunk 1]
```diff
@@ -141,18 +151,56 @@ def get_stats_index_hit_rates(db_connection):
...
+    ratio = members // MAX_MULTIXACT_MEMBERS
+    percentage_remaining = (1.0 - ratio) * 100
+    return [metric_multixact_members_remaining_ratio(percentage_remaining)]
+
+
+def get_multixact_remaining_ratio(_data_dir, db_connection):
+    mxid_age = get_max_mxid_age(db_connection)
+    if not mxid_age:
+        return []
+    ratio = mxid_age // WRAPAROUND_LIMIT
```
Comment (mahlberg) [inline postgresql_metrics/metrics_gatherer.py]: Use `/` instead of `//` here.

File: postgresql_metrics/metrics_gatherer.py [hunk 2]
```diff
@@ -141,18 +151,56 @@ def get_stats_index_hit_rates(db_connection):
...
+    percentage_remaining = (1.0 - ratio) * 100
+    return [metric_multixact_remaining_ratio(percentage_remaining)]
+
+
+def get_xid_remaining_ratio(_data_dir, db_connection):
+    xid_age = get_max_xid_age(db_connection)
+    if not xid_age:
+        return []
+    ratio = xid_age // WRAPAROUND_LIMIT
```
Comment (mahlberg) [inline postgresql_metrics/metrics_gatherer.py]: Use `/` instead of `//` here.

**Comment by mahlberg** [comment]
Tested manually on existing database instance, seems to work as expected :+1:

Noticed that documentation in README.md needs to be updated to reflect that "Database Local Directory Based (Global) Metrics" are now handled as "Database Cluster (Global) Metrics". Other than that it looks good to me.

**Comment by mahlberg** [comment]
LGTM 👍

**Review by dflemstr — APPROVED** [review]

**Pull request merged**